src/types.ts

```diff
@@ -20,19 +20,30 @@ import { IMocks } from 'graphql-tools'
 import { IMiddleware as IFieldMiddleware } from 'graphql-middleware'
 
 export type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
-export interface IResolvers {
-  [key: string]: (() => any) | IResolverObject | GraphQLScalarType
+
+export type IEnumResolver = { [key: string]: string | number };
```

Comment:
Suggest: `Record<string, string | number>`?

Reply:
That woulod be equivalent yes, just matching how [graphql-tools](https://github.com/Slessi/graphql-tools/blob/master/src/Interfaces.ts#L84) has it